src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/UserDefinedImplicitConversions.cs

```diff
@@ -642,6 +642,9 @@ private static bool IsEncompassingImplicitConversionKind(ConversionKind kind)
 
                 // Added for C# 7.1
                 case ConversionKind.DefaultLiteral:
+
+                // Added for C# 10.
+                case ConversionKind.FunctionType:
```

Comment:
>case ConversionKind.FunctionType: [](http://example.com/codeflow?start=16&length=33)

Follow up with language design: should a function type conversion be considered an "encompassing" implicit conversion?